enterprise/backend/src/metabase_enterprise/sso/integrations/saml.clj

```diff
@@ -236,11 +236,11 @@
     (walk/postwalk
      (fn [x]
        (when (and (map? x)
-                  (= (:tag x) :StatusCode)
+                  (= (:tag x) :samlp:StatusCode)
```

Comment:
is this legit? looks weird

Reply:
yeah new core.xml started producing namespaces >_>